solar_car_process.py

```diff
@@ -15,4 +22,25 @@ def start(self):
     
     def check_status(self):
         return self.process.poll()
+
+    
+class Domovoi:
+        
+    def run(self):
+        with open(sys.argv[1]) as file:
```

Comment:
Command line arguments should get passed into this method, so the file. This leaves it a little ambiguous what `sys.argv[1]` is.